src/ExceptionHandling.java

```diff
@@ -0,0 +1,43 @@
+/*
+ Developed by Ritabrata, 2024
+ */
+
+
+class performOperations {
+    public void performDivision(int a, int b) {
+        try {
+            System.out.println(a / b);
+        }
+        // Since in this case we know it will throw an Arithmetic exception in try block, so
+        // we are catching the exception through ArithmeticException
+        catch (ArithmeticException ae) {
+            System.out.println(ae.getMessage());
+        }
+        // This catch block will catch generic exception.
+        // It is used when we don't know the possible cause for exception
+        catch (Exception e) {
+            // getMessage() will only give the reason of exception without any class details or stackTrace
+            //System.out.println("Cannot perform division : " + e.getMessage());
```

Comment:
Is this supposed to be in the code or commented?

Reply:
It's a good practice not to keep commented code, but this is for reference purpose. We can keep as of now. In future, will create a PR template where we need to remove all commented code if it's not required.

Reply:
Okay